src/popups/PopupOptions.as

```diff
@@ -1019,6 +999,13 @@ package popups
                 box.addChild(gameNoteColorTitle);
                 yOff += 24;
 
+                // Create ComboBox Data
```

Comment:
Generally making the same change in two different branches is risky. Better bet would be to either not include the change or base your branch off of your feature branch, rebasing it after the previous submission.

Reply:
I see. I have removed the changes from my previous PR in this one.

Reply:
Sweet! This also helps for if this PR gets submitted before the other one, for whatever reason.